server/config/passport.js

```diff
@@ -108,14 +113,20 @@ passport.use(new GitHubStrategy({
         existingEmailUser.verified = User.EmailConfirmation.Verified;
         existingEmailUser.save(saveErr => done(null, existingEmailUser));
       } else {
-        const user = new User();
-        user.email = primaryEmail;
-        user.github = profile.id;
-        user.username = profile.username;
-        user.tokens.push({ kind: 'github', accessToken });
-        user.name = profile.displayName;
-        user.verified = User.EmailConfirmation.Verified;
-        user.save(saveErr => done(null, user));
+        let { username } = profile;
```

Comment:
I'm a bit confused about when this flow will be triggered? If the user does not have an Editor account with the same email as their Github account?

Reply:
It's expected that this would trigger when a user is registering for the Editor with a GitHub account, i.e. they clicked "Login with GitHub" from the Signup Page.

This just reminded me that I should test if linking social accounts works properly if the email they registered with for the Editor is different from the email for their GItHub/Google Account.